sw/device/lib/base/hardened_status.h

```diff
@@ -28,17 +28,23 @@ extern "C" {
 /**
  * Hardened version of the `TRY` macro from `status.h`.
  *
+ * Returns the error unmodified if `status_ok` fails, and throws an
+ * `INTERNAL()` errror if the OK code does not match the hardened value.
```

Comment:
typo "errror". we can fix it forward when convenient.

Reply:
Thanks for the catch! Fixed in https://github.com/lowRISC/opentitan/pull/18637